### ISSUE TITLE
Import e-mails

### ISSUE THREAD

**Issue by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1235]
Can user import e-mail`s conversation from mbox, MH, eml, msg and so on ?

After import, the tree structure must correspond to the structure of the correspondence (letter - reply - reply to reply).

And how user can do it ?

Or it need to write special plugin?

**Comment by jaap-karssenberg** [comment]
Yes you will need a plugin or script to turn email in wiki pages.

Op wo 19 aug. 2020 14:15 schreef Nick222 <notifications@github.com>:
